src/commands/support/winss.ts

```diff
@@ -0,0 +1,27 @@
+import { CommandDefinition } from '../../lib/command';
+import { makeEmbed, makeLines } from '../../lib/embed';
+import { CommandCategory } from '../../constants';
+
+const SNIPPING_TOOL_URL = 'https://media.discordapp.net/attachments/897491699167793182/1040649316672479232/image.png';
+
+export const winss: CommandDefinition = {
+    name: ['winss', 'sswin'],
```

Comment:
```suggestion
    name: ['winss', 'sswin'],
```

'howtoscreenshot' could be a nice alias too! (or 'howtoss')